Update Feature-Policy to Permissions-Policy (#22439)
mdn/content: files/en-us/web/api/navigator/getbattery/index.md

monitor the battery status. This implements the [Battery Status API](/en-US/docs/Web/API/Battery_Status_API); see that
documentation for additional details, a guide to using the API, and sample code.

> > **Note:** In some browsers access to this feature is controlled by the {{HTTPHeader("Permissions-Policy")}} directive {{HTTPHeader("Permissions-Policy/battery","battery")}}.

Reviewer: This is a future proof version
```suggestion
> **Note:** Access to this feature may be controlled by the {{HTTPHeader("Permissions-Policy")}} directive {{HTTPHeader("Permissions-Policy/battery","battery")}}.
```
